web/src/buddy_list.js

```diff
@@ -27,8 +29,10 @@ class BuddyListConf {
 
     get_li_from_key(opts) {
         const user_id = opts.key;
-        const $container = $(this.container_sel);
-        return $container.find(`${this.item_sel}[data-user-id='${CSS.escape(user_id)}']`);
+        const $narrow_users_container = $(this.narrow_container_sel);
+        return $narrow_users_container.find(
+            `${this.item_sel}[data-user-id='${CSS.escape(user_id)}']`,
+        );
     }
```

Comment:
I don't remember what this function does but it seems like something that should find things in `$other_users_container` too?